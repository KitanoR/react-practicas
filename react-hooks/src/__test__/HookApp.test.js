import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import { HookApp } from '../HookApp';

describe('Tests para el comonente de hook app', () => {
    test('Debe mostrar <Hookapp />', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
    test('Debe contener hola mundo', () => {
        const wrapper = shallow(<HookApp />);
        const h1 = wrapper.find('h1');
        expect(h1.text().trim()).toBe('Hola Mundo')
    });
});