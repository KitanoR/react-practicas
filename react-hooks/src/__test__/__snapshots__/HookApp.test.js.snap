// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests para el comonente de hook app Debe mostrar <Hookapp /> 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
